packages/dd-trace/src/proxy.js

```diff
@@ -217,8 +238,16 @@ class Tracer extends NoopProxy {
         const prioritySampler = appsecStandalone.configure(config)
         this._tracer = new DatadogTracer(config, prioritySampler)
         this.dataStreamsCheckpointer = this._tracer.dataStreamsCheckpointer
-        this.appsec = new AppsecSdk(this._tracer, config)
-        this.llmobs = new LLMObsSDK(this._tracer, this._modules.llmobs, config)
+        if (config.appsec.enabled) {
+          this.appsec = new lazyClasses.AppsecSdk(this._tracer, config)
+        } else {
+          this.appsec = new lazyClasses.NoopAppsecSdk()
+        }
```

Comment:
AppSec SDK needs to be available even if AppSec is disabled. I think this is breaking that

Reply:
(the noopAppsecSdk is used when tracing is disabled)